deploy/examples/direct-mount.yaml

```diff
@@ -24,12 +24,12 @@ spec:
           imagePullPolicy: IfNotPresent
           tty: true
           env:
-            - name: ROOK_CEPH_USERNAME
+            - name: ROOK_OPERATOR_CEPH_USERNAME
```

Comment:
I think it'll keep things simpler if we don't rename env vars. It's still just the ceph username that the operator code or toolbox is consuming. Or is there a need for a different var name?